ztag/transforms/https.py

```diff
@@ -2,6 +2,17 @@
 from ztag import protocols, errors
 from ztag.transform import Transformable
 from ztag.errors import IgnoreObject
+from http import HTTPTransform
+
+
+class HTTPSGetTransform(HTTPTransform):
+    name = "https/generic"
+    port = None
+    protocol = protocols.HTTPS
+    subprotocol = protocols.HTTPS.GET
```

Comment:
Is this from a protobuf definition? Have we added this subprotocol there (if necessary)?

Reply:
I was worried about that too, but I think we're good: if you check zsearch_definition's protocols.py, all of the subprotocols are registered with `setattr(Protocol, simple_name, object_)`, so the value is definitely there (since the Subprotocol `GET` already exists, being used by `HTTP`).

If there is someplace further down the data pipeline where specific `(Protocol, Subprotocol)` pairs are registered, there may be some issues there -- but I don't recall having any similar issues with e.g. using the `BANNER` subprotocol on the new database protocols.

Reply:
> If there is someplace further down the data pipeline where specific (Protocol, Subprotocol) pairs are registered, there may be some issues there

I don't know offhand whether or not this is the case ¯\\\_(ツ)\_\/¯